timescaledb/tutorials/analyze-intraday-stocks/explore-stocks-data.md

```diff
@@ -0,0 +1,284 @@
+# Explore stock market data
+
+When you've successfully collected 1-min intraday stock data, it's time to have some fun and explore the 
+data.
+
+Because of the high granularity of the dataset, there are numerous ways to explore it. For example, you could analyze stock prices and volumes on a minute-by-minute basis. With TimescaleDB, you could also bucket records into custom intervals (e.g. 2-min or 15-min) using TimescaleDB aggregate functions. 
+
+Let's see how it's done!
+
+## Install Plotly and Pandas
+
+To get started with data exploration, you need to install a couple of tools first:
+
+* Pandas, to query and structure the data
+* Plotly, to create visualizations quickly
+
+**Install both**
+```bash
+pip install plotly pandas
+```
+
+When you have those installed, you need to open a new Python file, or use a Jupyter notebook to 
+start exploring the dataset.
+
+## Establish database connection
+
+Use the configuration file you created earlier with `psycopg2` to create a database connection object:
+
+```python
+import config, psycopg2
+conn = psycopg2.connect(database=config.DB_NAME,
+                        host=config.DB_HOST,
+                        user=config.DB_USER,
+                        password=config.DB_PASS,
+                        port=config.DB_PORT)
+```
+
+In each data exploration script below, you need to reference this connection object to be able to 
+query the database.
+
+## Generate stock market insights
+
+Let's start off analyzing trading volumes, then have a look at weekly price points, and finally, dig deep on 
+price changes.
```

Comment:
Add to the end of this line:

"The results of the queries shown are visualized in (what visualization tool are you using?)."

Reply:
https://github.com/timescale/docs/pull/150/commits/6f26ecaec1da60757fc45c4c2dba2dd749700b25